src/crypto/store/indexeddb-crypto-store-backend.js

```diff
@@ -257,6 +257,38 @@ export class Backend {
         };
         return promiseifyTxn(txn);
     }
+
+    /**
+     * Load the end to end account for the logged-in user. Once the account
+     * is retrieved, the given function is executed and passed the base64
+     * encoded account string and a method for saving the account string
```

Comment:
s/base64 encoded/pickled/? And s/account string/pickle/?

Might be more consistent. just a thought.

Reply:
yeah, wfm